completions/drush.completion.sh

```diff
@@ -7,7 +7,7 @@
 #   http://github.com/drush-ops/drush/blob/master/drush.complete.sh
 
 # Ensure drush is available.
-which drush &> /dev/null || alias drush &> /dev/null || return
+_omb_util_command_exists drush &> /dev/null || alias drush &> /dev/null || return
```

Comment:
remove &>/dev/null

Reply:
We can actually also remove the check of `alias drush` because it is contained in `_omb_util_command_exists`.